README.md

```diff
@@ -46,6 +46,8 @@ You will need to be online to authenticate even when developing in localhost.
 - See `package.json` for npm package dependencies.
 - The web server used is the `express` web framework for node.js
 - React components are located in `src/components/` directory
+- Characters for the diacritics components come from the
+  [Mediawiki](https://github.com/wikimedia/mediawiki/) repository's [specialcharacters.json](https://github.com/wikimedia/mediawiki/blob/master/resources/src/mediawiki.language/specialcharacters.json). This file has been customized for use in Sinopia.
```

Comment:
Can you describe what you mean by "customized"?

Reply:
Done, added more context to the changes.